tensorflow/docs_src/programmers_guide/datasets.md

```diff
@@ -190,7 +190,7 @@ validation_dataset = tf.data.Dataset.range(50)
 # A reinitializable iterator is defined by its structure. We could use the
 # `output_types` and `output_shapes` properties of either `training_dataset`
 # or `validation_dataset` here, because they are compatible.
-iterator = Iterator.from_structure(training_dataset.output_types,
+iterator = tf.data.Iterator.from_structure(training_dataset.output_types,
                                    training_dataset.output_shapes)
```

Comment:
Can you please fix the indentation on this line to align with the `(` on the line above?

Reply:
I fixed the indentation according to your request to align with the **(** on the line above